Dockerfile

```diff
@@ -6,7 +6,7 @@ WORKDIR deck
 
 RUN docker/setup-apache2.sh
 
-RUN ./gradlew build -PskipTests
+RUN git config --global url.https://github.com/.insteadOf git://github.com/ && ./gradlew build -PskipTests
```

Comment:
Oops - it's a workaround for our security policies. We only allow https on corporate network. Removing.